src/Mvc/test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -44,6 +44,16 @@ public IActionResult DownloadFromDiskWithFileName_WithLastModifiedAndEtag()
         return PhysicalFile(path, "text/plain", "downloadName.txt", lastModified, entityTag, true);
     }
 
+    public IActionResult DownloadFromDiskSymlink()
+    {
+        var path = Path.Combine(_hostingEnvironment.ContentRootPath, "sample.txt");
+        var symlink = Path.Combine(_hostingEnvironment.ContentRootPath, Path.GetRandomFileName());
+
+        var fileInfo = System.IO.File.CreateSymbolicLink(symlink, path);
```

Comment:
@pranavkm , is ok to create files in the disk during the functional Tests or another approach must be uses?

Reply:
We generally avoid it but it might be ok for limited use. One thing I would avoid though is adding files to the project as part of running the test. This would result in new files being created in the repo as a result of running the tests since the test project is being run in-place. You could consider using the TempDir instead